docoptTests/DocoptTestCaseParser.swift

```diff
@@ -91,7 +96,13 @@ public struct DocoptTestCaseParser {
             NSLog("Error parsing \(expectedOutput) to JSON: \(error)")
             return nil
         }
-        let expectedOutputJSON: AnyObject? = NSJSONSerialization.JSONObjectWithData(jsonData!, options: .AllowFragments, error: &error)
+        let expectedOutputJSON: AnyObject?
```

Comment:
the do/catch lets us log reasonable output. what would you do in the else clause of guard? i haven't seen any examples where the method can throw (aka has NSError in/out).